### PR TITLE
Moved the instance quotas test to their own module, increased the instance count.

### PR DESCRIPTION
**Pull request by TimSimpsonR**
- This allows the quota test to work while not using low quotas which would break the pagination tests.

### CODE REVIEW

**Comment by rnirmal** [comment]
+1

**Comment by ed-** [comment]
+1 | cowsay | figlet

File: tests/integration/tests/api/instances_quotas.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+GROUP = "dbaas.api.instances.status"
```
Comment (cp16net) [inline tests/integration/tests/api/instances_quotas.py]: this looks like some excess baggage.

Comment (TimSimpsonR) [inline tests/integration/tests/api/instances_quotas.py]: Thanks. Eliminated.

**Comment by cp16net** [comment]
+1

**Comment by imsplitbit** [comment]
+1

**Comment by sacharya** [comment]
+1

**Pull request merged**